### PR TITLE
Post code review slump

### PR DESCRIPTION
**Pull request by kellymdev**

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def update
     if params[:start_game]
       StartGame.new(game: @game, player: @player).call
     elsif params[:roll_dice]
-      RollDice.new(game: @game, player: @player).call
+      PerformMove.new(game: @game, player: @player, dice_roll: RollDice.new.call).call
```
Comment (akampjes) [inline app/controllers/games_controller.rb]: MovePlayer.new...

File: app/presenters/properties_presenter.rb [hunk 2]
```diff
@@ -31,7 +31,7 @@ def owner(deed)
   end
 
   def current_rent(property)
-    CalculateRent.new(property: property, game: @game).call
+    CalculateRent.new(property: property, game: @game, dice_roll: 0).call
```
Comment (akampjes) [inline app/presenters/properties_presenter.rb]: Dice roll optional.

File: app/services/add_house_to_deed.rb [hunk 3]
```diff
@@ -39,30 +36,22 @@ def player_owns_set?
...
-    else
-      @deed == deed_for_next_house
-    end
+    @player.deeds.select { |deed| deed.property.colour == @deed.property.colour }
   end
 
-  def deeds_have_the_same_number_of_houses?
-    deeds_for_set.all? { |deed| deed.house_count == deeds_for_set.first.house_count }
+  def houses_being_added_evenly?
+    @deed.house_count + 1 <= lowest_house_count_for_set + 1
```
Comment (akampjes) [inline app/services/add_house_to_deed.rb]: - + 1

File: app/services/calculate_rent.rb [hunk 4]
```diff
@@ -1,8 +1,9 @@
 class CalculateRent
-  def initialize(game:, property:)
+  def initialize(game:, property:, dice_roll:)
```
Comment (akampjes) [inline app/services/calculate_rent.rb]: `dice_roll: 0`

File: app/services/calculate_rent.rb [hunk 5]
```diff
@@ -38,11 +29,18 @@ def deed
...
+      .includes(:deeds)
+      .where(category: Property::RAILROAD, deeds: {player: property_owner})
+      .count
   end
 
   def number_of_utilities
-    Property.includes(:deeds).where(propertytype: Property::UTILITY, deeds: {player: property_owner}).count
+    # TODO: check without includes
+    Property
+      .includes(:deeds)
```
Comment (akampjes) [inline app/services/calculate_rent.rb]: try with out `includes`

Comment (kellymdev) [inline app/services/calculate_rent.rb]: This fails without includes as it then can't find the deed.

**Pull request merged**